core/ui/public/i18n/en/translation.json

```
@@ -195,8 +195,12 @@
         "join_cluster_instead": "Join an existing cluster instead",
         "create_cluster_instead": "Create a new cluster instead",
         "join_code_helper_text": "You can obtain join code by accessing",
-        "invalid_join_code": "Invalid join code",
         "tls_verify": "TLS certificate validation",
+        "the_join_code_can_not_be_decoded": "The join code can not be decoded",
+        "the_join_code_is_not_correctly_encoded": "The join code is not correctly encoded",
+        "the_join_code_cannot_be_parsed": "The join code cannot be parsed",
+        "cluster_tls_verify_error": "The leader's TLS certificate is not valid",
+        "cluster_auth_error": "The join cluster answer is not valid",
```

```suggestion
        "cluster_auth_error": "Invalid response",
```
This is just a suggestion, when does this error occurs?